src/spikeinterface_pipelines/curation/curation.py

```diff
@@ -38,7 +38,7 @@ def curate(
         Curated sorting
     """
     # get quality metrics
-    if not waveform_extractor.has_extension("quality_metrics"):
+    if not waveform_extractor.is_extension("quality_metrics"):
```

Comment:
```suggestion
    if not waveform_extractor.has_extension("quality_metrics"):
```
with 0.100 this is the correct syntax